editions/tw5.com/tiddlers/wikitext/Wikitext Cheat-Sheet.tid

```diff
@@ -0,0 +1,143 @@
+created: 20220821184022101
+modified: 20221010130927409
+tags: WikiText
+title: Wikitext Cheat-Sheet
+type: text/vnd.tiddlywiki
+
+\define extLink(text tid) [ext[$text$|https://tiddlywiki.com/#$tid$]]
+\define link(text tid) [[$text$|$tid$]]
+
+\define exportFilter()
+[[$(me)$]] [tag[$(me)$]]
+\end
+
+\define exportMeButton(btnText:"Export Me & Tagged")
+\whitespace trim
+<$set name="me" value=<<currentTiddler>> >
+  <$tiddler tiddler="$:/core/templates/exporters/JsonFile">
+    <$button class="tc-btn-download">
+      <$action-sendmessage $message="tm-download-file" $param=<<currentTiddler>> exportFilter=<<exportFilter>> filename={{{ [<me>addsuffix{!!extension}] }}}/>
+      <$text text="""$btnText$"""/>
+    </$button>
+  </$tiddler>
+</$set>
+\end
+
+\define .xL(text tid)
+\whitespace trim
+<$set name="link" value={{{ [[$tid$]is[tiddler]then<link """$text$""" """$tid$""">else<extLink """$text$""" """$tid$""">] }}}>
+<<link>>
+</$set>
+\end
+
+\define .t(tid) <$transclude tiddler="""$tid$""" mode=block/>
+\define .cb(title) <$codeblock code={{$title$}}/>
+
+<!-- Helper macros. The "old" wikitext style is intentional. The code should be compatible with TW v5.1.15 -->
```

Comment:
I don't think that backwards compatibility should be a goal, not if it constrains the design to the extent that it has. Let's do it properly using modern features.